ts/indicators/indicators.src.ts

```diff
@@ -61,6 +61,12 @@ declare global {
             yData: Array<(number|null)>;
         }
 
+        interface IndicatorMultipleNullableValuesObject {
+            values: Array<Array<(number|null)>>;
+            xData: Array<(number|null)>;
+            yData: Array<Array<(number|null)>>;
+        }
+
         interface IndicatorMultipleValuesObject {
             values: IndicatorValuesObject['values'];
             xData: IndicatorValuesObject['xData'];
```

Comment:
For consistency you should inherit the type for `values` and `xData` in `IndicatorMultipleNullableValuesObject` the same way like in `IndicatorMultipleValuesObject` - or if you think it is wrong, instead replace the type inheritence in `IndicatorMultipleValuesObject`.

Reply:
Fixed as described first in your comment :)